### ISSUE TITLE
`Environment.getArgs` doesn't work when running with `run` command

### ISSUE THREAD

**Issue by mlutze** [flix/flix#5521]

**Comment by JonathanStarup** [comment]
In vscode or from terminal?

**Comment by mlutze** [comment]
From terminal

**Comment by mlutze** [comment]
It returns Nil

**Comment by JonathanStarup** [comment]
On another note, I didn't find it easy to figure out how to use the CLI.

I first tried this but I think something is being interpreted as a file maybe? but no error
```
java -jar flix.jar run something
```
This indeed does not seem to work
```
java -jar flix.jar run --args something
```

**Comment by JonathanStarup** [comment]
Fixed by #5778
